doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -98,25 +98,16 @@ PostgreSQL
 Installing PostgreSQL
 =======================
 
-Version 9.6 is strongly recommended because it is the version developers and QA test with::
+The application has been tested with PostgreSQL versions up to 13. We recommend installing the latest version that is available for your OS distribution. *For example*, to install PostgreSQL 13 under RHEL/CentOS 7::
```

Comment:
We should update the dev guide too. Right now it starts with, "PostgreSQL 9.6 is recommended to match the version in the Installation Guide."

I also think we should update Vagrant and docker-aio but especially since the Solr pull request touches both and hasn't been merged, it makes sense to wait.

Reply:
I'll definitely update the dev. guide, thanks. 
Still not sure about touching the vagrant and docker-aio in this PR. I'd rather have somebody who uses/works on those extensively handle that. And I don't see a problem with doing that separately/later. I mean, everything there is going to keep working just fine with PG 9.6; touching/changing anything there on the other hand comes with the risk of breaking something in that fragile ecosystem.